syncer/checkpoint.go

```diff
@@ -32,7 +32,7 @@ import (
 	"github.com/pingcap/dm/pkg/dumpling"
 	"github.com/pingcap/dm/pkg/gtid"
 	"github.com/pingcap/dm/pkg/log"
-	"github.com/pingcap/dm/pkg/schema"
+	schemapkg "github.com/pingcap/dm/pkg/schema"
```

Comment:
Why use package alias?

Reply:
Because we occasionally use `schema` as the variable name.